mailpoet/lib/Automation/Integrations/MailPoet/Payloads/CustomerPayload.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Payloads;
+
+use MailPoet\Automation\Engine\Integration\Payload;
+
+class CustomerPayload implements Payload {
+
+
+
+  private $customer;
```

Comment:
You seem to add more and more empty lines with every new class 😂

Could you also pls type the properties? Here and in all the other new classes? Otherwise, `$this->property` is not typed.

Reply:
:see_no_evil:  cleaned up.